### ISSUE TITLE
ZFS on Ubuntu Server Root documentation for SSH install.

### ISSUE THREAD

**Issue by YosuCadilla** [zfsonlinux/zfs#8208]
I found the existing articles about installing ZFS on Ubuntu using the desktop cd quite informative and interesting, however this is not really an option for dedicated servers, which probably have 2 or more disks that can be used in a mirror or stripped configuration where ZXS would really shine.

I would gladly help to produce such documentation and/or a video (once I know the proper way this has to be done).

A related questions would be, since Ubuntu is officially including ZFS and it's recommending its use for LXD, is there any information available on when they plan to include it in their installation processes/wizards?

**Comment by YosuCadilla** [comment]
Recommendations and best practices for common use cases could also a great resource.
I would also be very glad to produce the content if I can get someone to teach me and oversee the content I produce.

**Comment by YosuCadilla** [comment]
Thank you for the pointers @ghfields 
Indeed this is not about any bug, sorry. I'm used to other projects where we use the issues for all sort of topics, not just bugs.

I subscribed to the list yesterday, will post there as per your recommendation.

**Comment by phol** [comment]
To add to that, I think it would be great to also include the option to use the new ZFS 0.8 native encryption instead of LUKS for encrypting the root filesystem. It would remove the "number of moving parts" the setup has right now, however, I am not sure if zfs-initramfs supports unlocking at boot already? Personally I am also using dropbear with `cryptroot-unlock` to unlock over SSH.

I used a setup like described in the wiki for a while but I reverted to LUKS with ext4, since especially when using a lot of memory / performing a lot of IO operations, the system would lock and crash and performance was generally a lot worse than it was using ext4. Moreover, after unlocking, the filesystem would sometimes not mount.

I thought it might have to do with the complexity the setup had and would be curious if removing LUKS from the mix would resolve these issues..
